influencer-details.blade.php

```diff
@@ -78,11 +78,12 @@ <h2 class="influencer-name">{{influencer name here}}ريهانا</h2>
         <div class="influencers-view influencers-wrapper row center-sm">
           <div class="row center-sm col-xs-12 col-sm-9">
             <div class="row col-xs-12 col-sm-9">
+              @foreach ($content['articles'] as $article)
           <!-- single artilce -->
           <!-- you need to loop on this article only the rest can be deleted they are for demo purposes -->
               <div class="influencers-article col-xs-12">
               <!-- article slug in the href -->
-                <a class="row" href="/articles/{{article.slug}}">
+                <a class="row" href="/-/articles/{{ $article['slug'] }}">
```

Comment:
link should go to website and not data. See https://github.com/Trelllis/najem-seo-templates/issues/4